api/insight/apiroutes.go

```diff
@@ -472,7 +472,7 @@ func (c *insightApiContext) getTransactions(w http.ResponseWriter, r *http.Reque
 		}
 		addresses := []string{address}
 		rawTxs, recentTxs, err :=
-			c.BlockData.ChainDB.InsightAddressTransactions(addresses, int64(c.Status.GetHeight()-2))
+			c.BlockData.ChainDB.InsightAddressTransactions(addresses, int64(c.Status.Height-2))
```

Comment:
It's definitely a bug with `insightApiContext` never updates any of the status fields.  The idea is probably to have the `insightApiContext` have a pointer to the same Status in `appContext`.